### PR TITLE
[sdk-chroot] Rename mer-sdk-chroot to sdk-chroot. Fixes JB#56734

### PR DESCRIPTION
**Pull request by vigejolla**
The script still has several references to mer. Those will be handled in separate tasks.

### CODE REVIEW

**Comment by martyone** [comment]
I would prefer just "sdk-chroot".

**Comment by vigejolla** [comment]
> I would prefer just "sdk-chroot".

Fine by me.

File: sdk-setup/src/sdk-chroot [hunk 1]
```diff
@@ -1,19 +1,14 @@
...
 
 usage()
 {
     cat <<EOF
     usage: $0 [-u <user>] [-m <all|none|root|home>] [-r <SDK root path>] [<command> <args> ..]
            $0 -h
 
-       This is the Mer chroot SDK.
-       For information see https://sailfishos.org/wiki/Platform_SDK
+       This is the Sailfish OS chroot SDK.
```
Comment (martyone) [inline sdk-setup/src/sdk-chroot:10]: maybe

```suggestion
       This is the minimal, chroot-based Sailfish SDK.
```

**Review by martyone — APPROVED** [review]
One suggestion

File: sdk-setup/src/sdk-chroot (hunk 1 above)
Comment (Thaodan) [reply sdk-setup/src/sdk-chroot:10]: Also don't remove the link to the wiki.

**Comment by martyone** [comment]
Provide an alias with deprecation warning for convenience?

**Comment by martyone** [comment]
And PR title needs update.

**Comment by Thaodan** [comment]
> I would prefer just "sdk-chroot".

There's more than one sdk so sdk ambiguous imho especially if you put the script into path.

File: sdk-setup/src/sdk-chroot (hunk 1 above)
Comment (vigejolla) [reply sdk-setup/src/sdk-chroot:10]: > Also don't remove the link to the wiki.

I didn't remove the link. I replaced it with link to the new documentation site.

**Comment by vigejolla** [comment]
> There's more than one sdk so sdk ambiguous imho especially if you put the script into path.

We don't recommend putting the script to path, we recommend using an alias.

**Comment by vigejolla** [comment]
> Provide an alias with deprecation warning for convenience?

Sure, why not

**Comment by Thaodan** [comment]
> @vigejolla commented on this pull request.
> 
> 
> 
> 
>  usage()
>  {
>      cat <<EOF
>      usage: $0 [-u <user>] [-m <all|none|root|home>] [-r <SDK root path>]
> [<command> <args> ..] $0 -h
> 
> -       This is the Mer chroot SDK.
> -       For information see https://sailfishos.org/wiki/Platform_SDK
> +       This is the Sailfish OS chroot SDK.
> 
> > Also don't remove the link to the wiki.
> 
> I didn't remove the link. I replaced it with link to the new documentation
> site.
> 
Ah ok the diff was shown wrong then.

**Comment by vigejolla** [comment]
> Ah ok the diff was shown wrong then.

You didn't look at the whole diff, you looked only at the small section which was shown attached to martyone's comment. As martyone's comment was on line just above the new link, the new link was not show.

**Comment by martyone** [comment]
I was rather thinking of `ln -s sdk-chroot mer-sdk-chroot` and then `[[ $0 = *mer-sdk-chroot ]] && echo ...` inside sdk-chroot. ...so that possible scripts do not break for users immediately.

**Comment by vigejolla** [comment]
> I was rather thinking of `ln -s sdk-chroot mer-sdk-chroot` and then `[[ $0 = *mer-sdk-chroot ]] && echo ...` inside sdk-chroot. ...so that possible scripts do not break for users immediately.

I thought about that, but then I realized the users would quite likely just miss the warning and never update their scripts. It's better to fail fast with an error message.

**Review by martyone — APPROVED** [review]
LGTM, quoting the message would not hurt ;)

**Pull request merged**